### ISSUE TITLE
Can't access horizontal scrollbar if the grid is not scrolled to bottom. [Windows 11] [Firefox]

### ISSUE THREAD

**Issue by Clement-DG** [ag-grid/ag-grid#5150]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```
If the "Always show scrollbars" option is off (Settings > Accessibility > Visual Effects > Always show scrollbars) and the user uses Firefox,  the horizontal scrollbar cannot be accessed until the grid is completely scrolled to bottom. (See examples on page https://www.ag-grid.com/vue-data-grid/column-definitions/)

**Current behavior**
We can't use the horizontal scrollbar unless the grid is completely scrolled to bottom.

**Expected behavior**
The scrollbar should be accessible

**Please tell us about your environment:**

* **OS :** 
    * Windows 11 :
 ![image](https://user-images.githubusercontent.com/81183515/167662132-71f0abc3-c6a2-4d99-8f13-69792c4c696e.png)

* **AG Grid version:** 27.2.1

* **Browser:**

    * Firefox : 
![image](https://user-images.githubusercontent.com/81183515/167662534-f1884791-9f27-4148-992c-db2755f37e72.png)

**Comment by makinggoodsoftware** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by chefnoname** [comment]
Hello!

Thank you for reporting this - much appreciated!

Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference and description:

**AG-6758 [Regression] When using Firefox, the horizontal scrollbar doesn't appear when hovered**

We try to fix bugs from one release to the next, so this should be fixed in the next release or the one after if it was raised too close to the next release date.

You can see the next release date on the product pipeline page:
https://www.ag-grid.com/ag-grid-pipeline/

**Now that this issue is recorded in our backlog, we will soon close the GitHub issue because we will not update it when this bug is fixed in a future version of AG Grid. The best way to get an update on this item is to sign up for AG Grid new release notifications. Once a new AG Grid version is released, look for the item reference above (AG-6578) on the changelog page to see if it's resolved in that version:
https://www.ag-grid.com/ag-grid-changelog/**

Thanks again for bringing this up with us - we appreciate your time!

**Comment by makinggoodsoftware** [comment]
Hi,

In order to help us keep our issues queue clear, we are going to close this issue

The official way to check the status for an issue raised here is to check its AG-xxxx ID against our pipeline and changelog

…the pipeline (if not yet developed)

https://www.ag-grid.com/ag-grid-pipeline/

... or the changelog, (if the issue has been resolved)

https://www.ag-grid.com/ag-grid-changelog/

We keep issues raised in the pipeline through github open for a while so that the community can add some feedback to them, and close them when there is no activity for a while.

If you want to add further feedback to a closed ticket with this comment, we would recommend opening a new ticket as we don’t monitor closed issues.

**Comment by walkermalling** [comment]
@makinggoodsoftware @chefnoname 

This regression seems to go back to at least 20.2.0 (specifically Firefox).

Your changelog https://www.ag-grid.com/changelog/  shows issue 6758 was fixed in 28.1, but it doesn't look like prior versions have been patched.

**Comment by StephenCooper** [comment]
That is correct, we do not patch old versions but release fixes in new versions. 

> Your changelog https://www.ag-grid.com/changelog/ shows issue 6758 was fixed in 28.1, but it doesn't look like prior versions have been patched.
